teslatoken.py

```diff
@@ -39,33 +42,47 @@ def main():
 
     headers = {'client_id': client_data['OWNERAPI_CLIENT_ID']}
 
+    request_url = 'https://owner-api.teslamotors.com/oauth/token';
+    acquire_token = True
     if args.username and args.password:
         headers.update({'grant_type': 'password', 'email': args.username, 'password': args.password})
         headers.update({'client_secret': client_data['OWNERAPI_CLIENT_SECRET']})
+    elif args.username and args.credential_file:
+        with open(args.credential_file) as f:
+            credentials = f.readline().strip()
+        headers.update({'grant_type': 'password', 'email': args.username, 'password': credentials})
+        headers.update({'client_secret': client_data['OWNERAPI_CLIENT_SECRET']})
```

Comment:
These two lines are basically identical to row 48-49. Maybe have one if-statement such as`if args.username and (args.password or args.credential_file)` instead, and store correct password in a variable to use in the header.

Reply:
Thanks for the feedback, resolved all the comments.